python/tests/test_compare_conservation_laws_sbml.py

```diff
@@ -0,0 +1,141 @@
+import importlib
+import amici
+import os
+import sys
+import pytest
+import numpy as np
+import warnings
+
+
+@pytest.fixture
+def edata_fixture():
+    """edata is generated to test preequilibration"""
+    edata = amici.ExpData(2, 0, 0,
+                          np.array([0., 0.1, 0.2, 0.5, 1., 2., 5., 10.]))
+    edata.setObservedData([1.5] * 16)
+    edata.fixedParameters = np.array([3., 5.])
+    edata.fixedParametersPreequilibration = np.array([0., 5.])
+    edata.reinitializeFixedParameterInitialStates = True
+
+    return edata
+
+
+def generate_models():
+    # SBML model we want to import
+    sbml_file = os.path.join(os.path.dirname(__file__), '..',
+                             'examples', 'example_constant_species',
+                             'model_constant_species.xml')
+    sbml_importer = amici.SbmlImporter(sbml_file)
+
+    # Name of the model that will also be the name of the python module
+    model_name =  model_output_dir ='model_constant_species'
+    model_name_cl = model_output_dir_cl = 'model_constant_species_cl'
+
+    # Define constants, observables, sigmas
+    constant_parameters = ['init_substrate', 'init_enzyme']
+    observables = {
+        'observable_product': {'name': '', 'formula': 'product'},
+        'observable_substrate': {'name': '', 'formula': 'substrate'},
+    }
+    sigmas = {'observable_product': 1.0, 'observable_substrate': 1.0}
+
+    # wrap models with and without conservations laws
+    sbml_importer.sbml2amici(model_name_cl,
+                             model_output_dir_cl,
+                             observables=observables,
+                             constantParameters=constant_parameters,
```

Comment:
please use `constant_parameters` instead of `constantParameters` (will be deprecated at some point)

Reply:
done already ;)

Reply:
Oh I mean also as argument name for `sbml2amici`, `constantParameters` still exists for backwards compatibility.

Reply:
Got it a minute later... :D
Done locally, need to push